### DISCUSSION TITLE
Link logo to main domain?

### DISCUSSION THREAD

**Discussion by ilyadynin** [shuding/nextra#1155] [Q&A]
So I host Nextra under a sub-domain (https:/help.domain.com) and I inserted my companies logo into the header. Now I wanted it to link to the main domain of my project, but when I put` <a href=""></a>` or `<Link>` around it, it does not work, but continues to redirect to the head of the docs site. Is it possible to change this behavior?

**Comment by dimaMachina** [comment]
could you provide a reproduction?

**Comment by MikeCarbone** [comment]
Have you tried the `logoLink` property in your theme config?
https://nextra.site/docs/docs-theme/theme-configuration#logo
